allenact/algorithms/onpolicy_sync/runner.py

```diff
@@ -608,19 +626,26 @@ def start_test(
 
         return self.log_and_close(
             start_time_str=self.checkpoint_start_time_str(checkpoint_paths[0]),
+            checkpoint_parts=checkpoint_parts,
             nworkers=num_testers,
             test_steps=steps,
             metrics_file=metrics_file_path,
         )
 
-    @staticmethod
-    def checkpoint_start_time_str(checkpoint_file_name):
+    def checkpoint_start_time_str(self, checkpoint_file_name):
         parts = checkpoint_file_name.split(os.path.sep)
         assert len(parts) > 1, f"{checkpoint_file_name} is not a valid checkpoint path"
         start_time_str = parts[-2]
         get_logger().info(f"Using checkpoint start time {start_time_str}")
         return start_time_str
 
+    def checkpoint_log_folder_str(self, checkpoint_file_name):
```

Comment:
Save comment as above. If this is a `@staticmethod` then you could also potentially call this from `main.py` before having instantiated the runner.